scripts/add_content_to_db.py

```diff
@@ -70,17 +70,44 @@
     }
 
     with open(args.csv, mode="r", encoding="utf-8") as file:
+        reader = csv.DictReader(file)
+        tags = [val.upper() for row in reader for val in eval(row["tags"])]
+        tags_db = requests.get(tags_endpoint, headers=headers)
+        tags_map = {val["tag_name"]: val["tag_id"] for val in tags_db.json()}
+
+        tags_db = [val["tag_name"] for val in tags_db.json()]
+        tags = set(tags) - set(tags_db)
+        if len(tags) > 0:
+            print(f"Adding new tags: {tags}")
+        for tag in tags:
+            payload = {
+                "tag_name": tag,
+            }
+            response = requests.post(
+                tags_endpoint, json=payload, headers=headers, verify=False
+            )
+            print(f"Status Code: {response.status_code}, Response: {response.text}")
+
+            if "tag_id" not in response.json():
+                raise ValueError("Could not create tag: {tag}.")
+            response_json = response.json()
+            tags_map[response_json["tag_name"]] = response_json["tag_id"]
+        else:
+            print("No new tags to add.")
+        file.seek(0)
         reader = csv.DictReader(file)
         for row in reader:
             if args.language:
                 language = args.language
             else:
                 language = row["language"]
+            content_tags = [tags_map[val.upper()] for val in eval(row["tags"])]
```

Comment:
Related to above, if we assume `"tag1, tag2"` instead of `"['tag1', 'tag2']"`, we can use

```suggestion
            content_tags = [tags_map[val.strip().upper()] for val in row["tags"].split(",")]
```